integration/policy_test.go

```diff
@@ -236,18 +238,18 @@ func TestPolicyCreateStdin(t *testing.T) {
 			fmt.Println(err)
 		}
 	}
+	assert.Contains(t, out.String(), "was created.")
+	assert.Empty(t, stderr.String(), "STDERR should be empty")
+	assert.Equal(t, 0, exitcode, "EXITCODE is not the expected one")
 
 	policyID, err := getPolicyIdFromStdout(out.String())
-	assert.Nil(t, err)
+	if err != nil {
+		assert.FailNow(t, err.Error())
+	}
```

Comment:
why these changes?

Reply:
We want to fail fast when unable to extract policyID.  Subsequent calls require a valid policyID and will implicitly fail w/o them....